plugins/com.google.cloud.tools.eclipse.appengine.facets/META-INF/MANIFEST.MF

```diff
@@ -14,18 +14,23 @@ Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.2.5",
  org.eclipse.jst.j2ee,
  org.eclipse.jst.j2ee.web,
  org.eclipse.m2e.maven.runtime,
- org.eclipse.m2e.core;bundle-version="1.6.2"
-Import-Package: com.google.cloud.tools.eclipse.util,
+ org.eclipse.m2e.core;bundle-version="1.6.2",
```

Comment:
I don't think you should have this in both Require-Bundle and Import-Package below. See what happens if you remove this one.